pyproject.toml

```diff
@@ -7,6 +7,7 @@ package-mode = false
 [tool.poetry.dependencies]
 python = "^3.10"
 ops = "^2.17.0"
+tenacity = "*"
```

Comment:
Why is tenacity not pinned at least to a major version?

Reply:
I didn't pin it because I didn't consider it important enough. If there is a known breaking change (like, for example with `websockets`) or something is very important and it's better to be cautious (like, e.g. `ops`), it makes sense to pin a version. If not, I prefer to fail fast :)